src/collections/members/divyangi-raghav/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+name: Divyangi Raghav 
+position: Community Manager
+image_path: ./divyangi-raghav.jpg
+github: divz-codess
+twitter: DivzzTwt
+linkedin: divyangi-raghav
+badges:
+ - layer5
+ - meshery
+ - smp
```

Comment:
```suggestion
 - community
```

Reply:
Psst. Community Manager @divz-codess, read these - https://layer5.io/community/handbook/recognition#Badges and Community Manager Trainer @Chadha93, ensure this is included as required reading.

Reply:
Done @leecalcote

Reply:
Sure, @leecalcote

Reply:
@divz-codess 
Please remove these badges:
 - layer5
 - meshery
 - smp
 
and let's update the position - community manager after the official announcement. You can add it as Contributor.

Reply:
Yes, take reference: https://github.com/layer5io/layer5/pull/3925